docs/concepts/collections.md

```diff
@@ -25,6 +25,31 @@ my_collection.ask("Find me Italian recipes for soups")
 
 In this scenario, the LLM first determines the most suitable view to address the query, and then that view is used to pull the relevant data.
 
+Sometimes, the selected view may not be able to answer the question and will raise an error. In such situations, the fallback collections can be used. This will cause a next view selection, but from the fallback collection.
```

Comment:
Just to bring my up to speed - why switching to a different collection when encountering errors in view execution? For me, the more intuitive solution would be to retry with the view, while providing the error information to the LLM. When I heard about "fallback collections" I thought they would be for cases when the view selector is not able to find any appropriate view in the origin collection, not for handling errors connected to individual views.

Reply:
We assume that llm pick wrong view for the ask therefore we want to look for another view. The idea is that in second collection would be freeform views that will operate as fallbacks.  When it will be in one collection just blacklisting checked first pick we are limited only to llm choice. In implemented case we can easily manage that second collection would have only freeform views.

Reply:
Ok. For me probably the jump from "error in a view" to "llm picked the wrong view" is non-obvious.